src/nxt_var.c

```diff
@@ -560,3 +571,18 @@ nxt_var_interpreter(nxt_task_t *task, nxt_tstr_state_t *state,
 
     return NXT_OK;
 }
+
+
+nxt_str_t *
+nxt_var_get(nxt_task_t *task, nxt_tstr_state_t *state, nxt_var_cache_t *cache,
+    nxt_str_t *name, void *ctx)
+{
+    nxt_var_ref_t  *ref;
+
+    ref = nxt_var_ref_get(state, name, cache->pool);
+    if (nxt_slow_path(ref == NULL)) {
+        return NULL;
+    }
+
+    return nxt_var_cache_value(task, state, cache, ref, ctx);
+}
```

Comment:
Personally I would add this in a separate commit, but it's small enough that I'll leave it up to you...